ste/sender-blockBlobFromURL.go

```diff
@@ -72,7 +72,7 @@ func (c *urlToBlockBlobCopier) GenerateCopyFunc(id common.ChunkID, blockIndex in
 	 * for blobs of all sizes.
 	 */
 	// Small blobs from all sources will be copied over to destination using PutBlobFromUrl
-	if c.NumChunks() == 1 && adjustedChunkSize <= int64(blockblob.MaxUploadBlobBytes) {
+	if c.NumChunks() == 1 && adjustedChunkSize <= int64(common.MaxBlockBlobBlockSize) {
```

Comment:
What API is this gating and what is its limit?

Reply:
This is gating the PutBlobURL API and the limit is 5000. Thanks for the callout, I will update this to use common.MaxPutBlobSize.